lib/streamlit/type_util.py

```diff
@@ -808,6 +808,25 @@ def is_pyarrow_version_less_than(v: str) -> bool:
     return version.parse(pa.__version__) < version.parse(v)
 
 
+def is_altair_version_less_than(v: str) -> bool:
```

Comment:
nit: we should consider having a small unit test for this convenience function with different inputs to ensure the behavior we intend it to have

Reply:
Hmm, I think it's not super easy to reasonably test this function. We could mock the altair version, but I don't think it would provide too much benefit. Also, the function will be used to skip quite a few unit tests that only work with < or > 5.x, so its passively tested that way.

Reply:
I added a dedicated `is_version_less_than` function with a test